server/chat/background/task.py

```
@@ -1864,6 +1868,101 @@ def _get_incident_data(incident_id: str, user_id: str) -> Optional[Dict[str, Any
+            'status': status,
+            'error': error_message,
+            'started_at': started_at,
+            'completed_at': datetime.now(timezone.utc),
+            'session_id': session_id,
+        }
+
+        # Send to primary + verified additional emails
+        all_emails = [user_email]
+        try:
+            with db_pool.get_admin_connection() as conn:
+                with conn.cursor() as cur:
```

Ruff (0.15.13)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)